### PR TITLE
Tmp Output F304 merged optimized CGM in RaoResult

### PR DESCRIPTION
**Pull request by pengbo-ruidian**
temporary solution for Rao integration.

### CODE REVIEW

File: ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/SearchTreeRao.java [hunk 1]
```diff
@@ -221,6 +221,9 @@ private RaoResult buildOutput() {
         RaoResult raoResult = new RaoResult(optimalLeaf.getStatus().equals(Leaf.Status.ERROR) ? RaoResult.Status.FAILURE : RaoResult.Status.SUCCESS);
         raoResult.setPreOptimVariantId(rootLeaf.getInitialVariantId());
         raoResult.setPostOptimVariantId(optimalLeaf.getBestVariantId());
+        if (raoResult.isSuccessful()) {
+            raoResult.setOptimizedNetwork(optimalLeaf.getRaoData().getNetwork()); //? TODO does the network in optimialLeaf contain all network actions ?
```
Comment (alexandrem-rte) [inline ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/SearchTreeRao.java:225]: No, you must add optimalLeaf.applyRangeActionResultsOnNetwork()

Comment (pengbo-ruidian) [reply ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/SearchTreeRao.java:225]: OK, could you please make a correction? I will just close this PR. thanks.

**Comment by pengbo-ruidian** [comment]
OK, could you please make a correction? I will just close this PR. thanks.

File: ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/SearchTreeRao.java (hunk 1 above)
Comment (pengbo-ruidian) [reply ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/SearchTreeRao.java:225]: I am not sure applyRangeActionResultsOnNetwork will solve the problem.

**Pull request closed without merging**